File: aula6/valendonota/exer1.py
#!/usr/bin/env python
now = 0
max = 0
cres = True
media = 0
meses = int(input("digite a quantidade de índices de audiência serão digitados"))
valores = []

for i in range(meses):
    now = float(input(" "))
    if now > max:
        max = now
    else:
       cres = False
    valores.append(now)

for i in valores:
    media += i

if cres == False:
    print("Média nem sempre crescente")
else:
    print("Média sempre crescente")
print("%.1f" % (media / meses))

File: aula6/valendonota/exer1.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
int main(){

 float now, max, media;
 float meses;
 int posArr = 0;
 bool cres = true;
 int vals[12] = {}; //não sei porque mas é necessário add um valor max na array

 printf("Digite a quantidade de indices de audiencia que serao digitados");
 scanf("%f", &meses);

 for(int i = 0; i < meses; i++){
   scanf("%f", &now);
   if(now > max)
     max = now;

   else
     cres = false;

   vals[posArr] = now;
   posArr = posArr + 1;

 }

 for(int l = 0; l <= posArr; l++){
  printf("\n %d l ", l);
  media += vals[l];
 }
float op = (media/meses); //por algum motivo a operacao dentro do print estava as vezes dando valores errados
 if(cres == false)
   printf("Média nem sempre crescente. %.1f", op);
 else
  printf("Media sempre crescente. %.1f", op);

 }
